js/main.js

```diff
@@ -0,0 +1,115 @@
+
+//The book class
```

Comment:
Remove all comments starting with this one

Reply:
Okey, commets deleted